web/ui/module/codemirror-promql/src/grammar/parser.js

```diff
@@ -0,0 +1,19 @@
+// This file was generated by lezer-generator. You probably shouldn't edit it.
```

Comment:
can you please remove this file.

It's not supposed to be versioned.

Reply:
Thanks, good catch.